File: Fixed Heap Dynamic/FHD.py
def fixed_heap_dynamic():
    import array
    arr = array.array('i', [0, 0, 0, 0, 0])
    for i in range(len(arr)):
        arr[i] = i * 5
    print("\nFixed Heap-Dynamic Array (array module):")
    print(arr.tolist())
fixed_heap_dynamic()

File: Fixed Heap Dynamic/FHD.java
public class FHD {
    public static void main(String[] args) {
        int[] arr = new int[7];
        for (int i = 0; i < arr.length; i++) {
            arr[i] = i * 5;
        }
        System.out.println("\nFixed Heap-Dynamic Array:");
        for (int num : arr) {
            System.out.print(num + " ");
        }
    }
}
